scout/adapter/mongo/query.py

```diff
@@ -248,9 +247,19 @@ def build_query(
                 continue
 
             if criterion == "chrom" and query.get("chrom"):  # filter by coordinates
+                query_chrom = query.get("chrom")
+                if isinstance(query_chrom, list):
```

Comment:
Wouldn't be easier to just pass a list here, also when there is only one chromosome?

Reply:
It would. The only thing is Im reluctant to put in the work to find all places it is called. Maybe I could use like pydantic to recast it. 😁

Reply:
> Maybe I could use like pydantic to recast it.

You could ask @mikaell that is very familiar with the library 🤣

Reply:
If you don't mind I will leave it for another time. I reread the criterion, and it would only shorten that very slightly, as there still is a need for different coordinate handling for SNV and SV on one hand, and for single and multiple chromosomes, with the latter not consistent with start-end coordinates. Looking at it again, I realised we were missing "cancer" in there, so they were always going as SV-overlaps. Should improve performance a little!

Reply:
To justify the laziness, the places to look for changes is not trivial. It's not just the forms, its all variants queries e.g. https://github.com/Clinical-Genomics/scout/blob/92753c7a07b0d2c5602a327bbf7d8541c00e358c/scout/server/blueprints/cases/controllers.py#L484). Also stored filters would be affected and would probably need resetting or script updating.

Reply:
I don't mind! 😆